Makefile

```diff
@@ -31,14 +33,26 @@ $(OPENAPI_FILE)::
 build:: $(OPENAPI_FILE)
 	$(GO) install $(VERSION_FLAGS) $(PROJECT)/cmd/$(PROVIDER)
 	$(GO) install $(VERSION_FLAGS) $(PROJECT)/cmd/$(CODEGEN)
-	for LANGUAGE in "nodejs" ; do \
-		$(CODEGEN) $(OPENAPI_FILE) pkg/gen/node-templates $(PACKDIR)/$$LANGUAGE || exit 3 ; \
+	for LANGUAGE in "python" ; do \
```

Comment:
Shouldn't this be `for LANGUAGE in "nodejs" "python" ; do \`? In other words, we want to generate both the Node.js package _and_ Python, not just one of them.